src/net/sourceforge/kolmafia/textui/command/AprilBandCommand.java

```diff
@@ -0,0 +1,161 @@
+package net.sourceforge.kolmafia.textui.command;
+
+import net.sourceforge.kolmafia.KoLCharacter;
+import net.sourceforge.kolmafia.KoLConstants.MafiaState;
+import net.sourceforge.kolmafia.KoLmafia;
+import net.sourceforge.kolmafia.RequestThread;
+import net.sourceforge.kolmafia.objectpool.ItemPool;
+import net.sourceforge.kolmafia.preferences.Preferences;
+import net.sourceforge.kolmafia.request.GenericRequest;
+import net.sourceforge.kolmafia.session.ChoiceManager;
+import net.sourceforge.kolmafia.session.InventoryManager;
+
+public class AprilBandCommand extends AbstractCommand {
+  public AprilBandCommand() {
+    this.usage =
+        "effect [nc|c|drop] | item [instrument] | play [instrument] - participate in the apriling band";
```

Comment:
Apriling band?  My sense is that Apriling is always used as a proper noun and hence capitalized.  Non-blocking.

Reply:
from looking at other files it seemed the standard is that instructions are always lowercase.

I'm not sure this is always, or even often true.

Reply:
OK.  I was looking at the failure message but...